libs/application-generic/src/usecases/get-subscriber-global-preference/get-subscriber-global-preference.usecase.ts

```diff
@@ -78,6 +71,15 @@ export class GetSubscriberGlobalPreference {
       subscriberGlobalChannels = subscriberGlobalPreferenceV2;
       enabled = true;
     } else {
+      // Lookup V1 preferences only if V2 is not available
```

Comment:
We save an extra DB call in some cases now (but all cases after migration) when V2 Preferences exist.